pkg/util/error_list.go

```diff
@@ -52,3 +57,62 @@ func SliceToError(errs []error) error {
 	}
 	return errorList(errs)
 }
+
+// Errors represents an object that may be transformed into a slice of errors by virtue
+// of being an error aggregate (a set of errors grouped together but not part of a parent
+// child relationship).
+type Errors interface {
```

Comment:
As far as I can tell, this is only used in this file - why can't you just try to convert to errorList or even []error?